resolver.go

```diff
@@ -184,7 +189,14 @@ func createRespMsg(query *dns.Msg) *dns.Msg {
 }
 
 func (r *resolver) handleIPQuery(name string, query *dns.Msg, ipType int) (*dns.Msg, error) {
-	addr, ipv6Miss := r.sb.ResolveName(name, ipType)
+	var addr []net.IP
+	var ipv6Miss bool
+	if r.sb != nil {
+		addr, ipv6Miss = r.sb.ResolveName(name, ipType)
+	} else if r.network != nil {
```

Comment:
We do have a slight difference in implementation in network and sandbox case though. In case of networks we want to reply back with a negative response if the query fails while in case of sandbox the resolver continues executing the query from the context of the sandbox (this functionality cannot be implemented in windows at this time)